src/TurbulenceConvection/types.jl

```diff
@@ -1,3 +1,5 @@
+include("../model_getters.jl")
+import ClimaAtmos.TurbulenceConvection.Parameters as TCP
```

Comment:
```suggestion
import .Parameters as TCP
```
I think this should work, we shouldn't ever need / use `import ClimaAtmos.anything` inside `src/` of `ClimaAtmos`. It technically opens the door to use multiple versions of ClimaAtmos.

Reply:
(same elsewhere)